packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -282,6 +310,58 @@ exports.appellantSubmissionDeclaration = async (req, res) => {
 	});
 };
 
+/**
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+exports.appellantSubmissionInformation = async (req, res) => {
```

Comment:
can we put the shared logic into a function and just not use the status bits?

Reply:
Good call, I've made the status param optional (with a default of an empty string) in the buildSectionViewModel function

Reply:
I meant the shared logic between list and appellantSubmissionInformation but it's not a major issue so not to worry